www/templates/www/base.html

```diff
@@ -3,93 +3,125 @@
 {% load static %}
 
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="robots" content="noindex, nofollow, noarchive" />
-    <title>{% block title %}{{ SITENAME }}{% endblock %}</title>
-    {% bootstrap_css %}
-    {% bootstrap_javascript jquery='full' %}
-    <link rel="shortcut icon" type="image/png" href="{% static 'www/favicon.ico' %}"/>
-    <link rel="stylesheet" href="{% static 'www/mulysa.css' %}" />
-    <script src="{% url 'javascript-catalog' %}"></script>
-  </head>
-  <body>
 
-    <header>
-      <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
-        <a class="navbar-brand" href="{% url 'index' %}">{{ SITENAME }}</a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta name="robots" content="noindex, nofollow, noarchive" />
+  <title>{% block title %}{{ SITENAME }}{% endblock %}</title>
+  {% bootstrap_css %}
+  {% bootstrap_javascript jquery='full' %}
+  <link rel="shortcut icon" type="image/png" href="{% static 'www/favicon.ico' %}" />
+  <link rel="stylesheet" href="{% static 'www/mulysa.css' %}" />
+  <script src="{% url 'javascript-catalog' %}"></script>
+</head>
 
-        <div class="collapse navbar-collapse" id="navbarCollapse">
-          <ul class="navbar-nav mr-auto">
+<style>
+  @media (min-width: 768px) and (max-width: 991px) {
+    .navbar-nav {
+      flex-wrap: wrap;
+    }
+
+    .navbar-nav .nav-item {
+      margin: 2px 0;
+    }
+  }
+
+  .navbar-collapse {
+    overflow-x: auto;
+    white-space: nowrap;
+  }
+
+  @media (max-width: 1199.98px) {
+    .priority-low {
+      display: none !important;
+    }
+  }
+</style>
```

Comment:
inline css is blocked in production because of the csp rules in place. maybe move the css into its own file ./www/static/www/mulysa.css